python/lsst/jointcal/jointcal.py

```diff
@@ -783,12 +788,12 @@ def _load_data(self, inputSourceTableVisit, inputVisitSummary, associations, joi
 
         return oldWcsList, bands
 
-    def _make_one_input_data(self, visitRecord, catalog):
+    def _make_one_input_data(self, visitRecord, catalog, detectorDict):
         """Return a data structure for this detector+visit."""
```

Comment:
Now that this is more than a completely trivial function, I think we probably do need a `Parameters` docstring.

Although, thinking about it, this should probably just be a closure inside `_load_data()`, so maybe it doesn't matter at all.

Probably nevermind me?